script/validate-content-builds.js

```diff
@@ -0,0 +1,14 @@
+const { runCommandSync } = require('./utils');
+
+// Run the vets-website content build
+runCommandSync('npm run build:content --pull-drupal');
+
+// Copy the drupal cache into the content-build repo
+runCommandSync('cp -r .cache ../content-build/.cache');
+
+// Run the standalone content build using the same cache
+runCommandSync('cd ../content-build && npm run build:content');
+
+// Compare the build outputs to see if they match
+const exitCode = runCommandSync('npm run build:compare');
```

Comment:
TIL about the `build:compare` script: 
https://github.com/department-of-veterans-affairs/vets-website/blob/751d6fa6fcf2974433475c96e4a660b1977ea98f/package.json#L10

Reply:
That was the "manual" script I wrote for doing this. It required a bit of setup to work right, such as running the builds yourself and hoping that they had the same content to begin with. This script eliminates those extra steps so it can be more automated.

Reply:
The `build:compare` script isn't meant to exist anymore once we finish with the content-build transition.